apps/src/sites/studio/pages/googleBlocklyWrapper.js

```diff
@@ -87,13 +86,34 @@ function initializeBlocklyWrapper(blocklyInstance) {
   blocklyWrapper.wrapReadOnlyProperty('Procedures');
   blocklyWrapper.wrapReadOnlyProperty('removeChangeListener');
   blocklyWrapper.wrapReadOnlyProperty('RTL');
+  blocklyWrapper.wrapReadOnlyProperty('selected');
```

Comment:
This is not really related to FieldDropdowns, just a property I forgot to wrap in the first PR, since it's not used in apps code anywhere. It's very useful during development though:
![image](https://user-images.githubusercontent.com/8787187/91897766-a26f5c00-ec4f-11ea-9c75-b64107f57cc7.png)